.github/workflows/cypress_workflow.yml

```diff
@@ -146,70 +203,93 @@ jobs:
       - run: npx cypress cache list
       - run: npx cypress cache path
 
-      # Setup spec files for Dashboards in-house cypress tests
-      - name: Setup spec files for Dashboards tests
-        if: ${{ inputs.specs == '' && matrix.group > 9 }}
+      # Run tests based on configuration
+      - name: Run FT repo tests with no query enhancements
+        if: matrix.test_location == 'ftr' && matrix.config == 'standard'
+        uses: cypress-io/github-action@v2
```

Comment:
did we want to bump the cypress github action if available and works? might have improvements https://github.com/cypress-io/github-action/blob/master/CHANGELOG.md. but i see somethings in our file will need to change so maybe not worht it at this time